Assets/Editor/GroundhogEditorMenu.cs

```diff
@@ -10,4 +11,70 @@ static void MovePlayerHere()
     {
         GameObject.FindWithTag("Player").transform.position = SceneView.lastActiveSceneView.camera.transform.position;
     }
+
+     [MenuItem("Assets/Create Reversed Clip", false, 14)]
```

Comment:
What does this do?

Reply:
This allows the user to access the function by right-clicking an item in the project window.